gensrc/script/functions.py

```diff
@@ -328,8 +328,8 @@
      'StringFunctions::translate_prepare', 'StringFunctions::translate_close'],
     [30431, 'crc32', 'BIGINT', ['VARCHAR'], 'StringFunctions::crc32'],
 
-    [30440, 'ngram_search', 'FLOAT', ['VARCHAR', 'VARCHAR'], 'StringFunctions::ngram_search','StringFunctions::ngram_search_prepare','StringFunctions::ngram_search_close'],
```

Comment:
If the return type is changed from float to double, will there be any compatibility issues during the grayscale upgrade?

Reply:
this function is not released yet